src/xenia/kernel/xam/xam_avatar.cc

```diff
@@ -17,24 +17,288 @@ namespace xe {
+                                                 dword_t unk5, qword_t unk6) {
+  // Function_818D1738(0x600004,param_6,&uStack_d0,0x90,0)
+
+  XELOGD("Stubbed");
+  return X_E_ACCESS_DENIED;
+}
+DECLARE_XAM_EXPORT1(XamAvatarGetManifestsByXuid, kAvatars, kStub)
+
+dword_result_t XamAvatarGetAssetsResultSize_entry(qword_t unk1, qword_t unk2,
+                                                  dword_t unk3) {
+  // return XMsgInProcessCall(0xf3,0x600005,local_20,0);
+
+  XELOGD("Stubbed");
+  return X_STATUS_SUCCESS;
+}
+DECLARE_XAM_EXPORT1(XamAvatarGetAssetsResultSize, kAvatars, kStub);
+
+dword_result_t XamAvatarGetAssets_entry(qword_t unk1, word_t unk2, dword_t unk3,
+                                        dword_t unk4, dword_t unk5,
+                                        qword_t unk6) {
+  XELOGD("Stubbed");
+  return X_STATUS_SUCCESS;
+}
+DECLARE_XAM_EXPORT1(XamAvatarGetAssets, kAvatars, kStub);
+
+dword_result_t XamAvatarSetCustomAsset_entry(qword_t unk1, qword_t unk2,
+                                             dword_t unk3, dword_t unk4,
+                                             dword_t unk5, dword_t unk6) {
+  // XMsgInProcessCall(0xf3,0x60000a,local_30,0);
+
+  XELOGD("Stubbed");
+  return X_STATUS_SUCCESS;
+}
+DECLARE_XAM_EXPORT1(XamAvatarSetCustomAsset, kAvatars, kStub)
+
+dword_result_t XamAvatarSetManifest_entry(qword_t unk1, qword_t unk2,
+                                          qword_t unk3) {
+  // Function_818D1738(0x600009,param_3,&local_410,0x3ec,0);
+
+  XELOGD("Stubbed");
+  return X_STATUS_SUCCESS;
+}
+DECLARE_XAM_EXPORT1(XamAvatarSetManifest, kAvatars, kStub);
+
+dword_result_t XamAvatarGetMetadataRandom_entry(qword_t body_type, qword_t unk2,
+                                                dword_t avatar_metadata,
+                                                qword_t unk4) {
+  // some_function(0x600010,param_4,local_28,0xc,local_30)
+
+  XELOGD("Stubbed");
+  return X_STATUS_SUCCESS;
+}
+DECLARE_XAM_EXPORT1(XamAvatarGetMetadataRandom, kAvatars, kStub);
+
+dword_result_t XamAvatarGetMetadataSignedOutProfileCount_entry(dword_t unk1,
+                                                               qword_t unk2) {
+  // Function_818D1738(0x600013,param_2,local_10,4,0);
+
+  XELOGD("Stubbed");
+  return X_STATUS_SUCCESS;
+}
+DECLARE_XAM_EXPORT1(XamAvatarGetMetadataSignedOutProfileCount, kAvatars, kStub);
+
+dword_result_t XamAvatarGetMetadataSignedOutProfile_entry(dword_t unk1,
+                                                          dword_t unk2,
+                                                          qword_t unk3) {
+  // Function_818D1738(0x600014,param_3,local_10,8,0);
+
+  XELOGD("Stubbed");
+  return X_STATUS_SUCCESS;
+}
+DECLARE_XAM_EXPORT1(XamAvatarGetMetadataSignedOutProfile, kAvatars, kStub);
+
+dword_result_t XamAvatarManifestGetBodyType_entry(qword_t body_type) {
+  // return either char of 1 - male, 2 - female, else - unknown
+
+  XELOGD("Stubbed");
+  return '\x01';
```

Comment:
Char of one like 1 in form of integer or 1 as a character?
For integer just write return 1;
For char write return '1';